### DISCUSSION TITLE
IND does not allow representation of unemployment in millions

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1381] [Indices and Indicators (IND)]
It only allows a percentage.
Not only are the numbers *published* with actual numbers, but that is frequently used in economic discussions.
For example from https://tradingeconomics.com/united-states/unemployment-rate "The US unemployment rate held steady at 3.5 percent in December 2019, remaining at the lowest level since 1969 and in line with market expectations. The number of unemployed people decreased by 58 thousand to 5.75 million while employment rose by 267 thousand to 158.80 million. "

The ontology does not allow you represent the 5.75 million.
This was previously discussed in issue #62 - it was requested there that a new issue be raised.

**Comment by ElisaKendall** [comment]
The unemployment rate is defined as having an index value which is a ratio value; a ratio value has a number associated with it, not a percentage, so I'm confused about what's missing here.  At the higher level, all economic indicators have an indicator value that is a quantity value ...

**Comment by rivettp** [comment]
We cannot represent the sentence I quoted "The number of unemployed people decreased by 58 thousand to 5.75 million while employment rose by 267 thousand to 158.80 million.". While not the "unemployment rate" it's important and distinct economic indicator.

**Comment by ElisaKendall** [comment]
So, we can represent the 'unemployed population', that has a universe size in non-negative integer,  although maybe that should be non-negative number, or decimal, for 5.78 -- we don't have the magnitude in millions or thousands, though, and it should be of type decimal (quantity value rather than nonNegativeInteger is needed there.  But we don't have the decrease or increase as a comparative number in that intervening time ... is that what you are asking for?

**Comment by rivettp** [comment]
If we have the unemployed population (also the employed population) of the US (or a State) with an associated dateTime that's indeed what I was looking for. People can calculate differences themselves.

**Comment by ElisaKendall** [comment]
In the EconomicIndicators ontology, we have fibo-ind-ei-ei;UnemployedPopulation, which is a subclass of fibo-ind-ei-ei;CivilianLaborForce.  This class does not represent the complete picture, as there are many people that are no longer considered as part of the unemployed population because they are not counted as looking for work.  However, the class fibo-ind-ei-ei;UnemployedPopulation corresponds exactly to the number of unemployed people reported by the Bureau of Labor Statistics in the US.  

We also have a class for fibo-ind-ei-ei;EmployedPopulation, again corresponding to the definition that BLS uses when publishing numbers.  The definition is the same one used by Canada.  And we have specializations for related concepts in the jurisdiction-specific ontologies.

There are additional classes that define things like the fibo-ind-ei-ei;CivilianNonInstitutionalPopulation, representing people that could be part of the labor force because they are able, but not necessarily considered part of the fibo-ind-ei-ei;CivilianLaborForce at some point in time.  

These are actually all statistical populations, which have a property called hasUniverseSize, which is a non-negative integer value - not a decimal with some qualifier, such as 'in millions'.  We could revise the Analytics ontology in FND to make this a quantity value and allow for representation of the number as a decimal with some qualifier, but the basic elements for representing the number of unemployed persons are there.

**Comment by rivettp** [comment]
Note my last comment included "with an associated dateTime". hasUniverseSize is a simple property and does not have this. Nor is there AFAIK a way of having different individuals of class UnemployedPopulation each representing USUnemployedPopulation and with different dateTimes. 
As ever, examples would help when asserting something can be represented.

**Comment by ElisaKendall** [comment]
You realize I assume that one does not need a restriction on a class to add property values to individuals, so what you are saying is that there are no properties that can support creating such individuals?

**Comment by rivettp** [comment]
It's not obvious. And if it's not to me, I don't think it will be to others. Rather than leaving everyone to invent something for themselves we should provide a best practice example as we have I think done for the UnemploymentRate. And that allows people to see the Rate alongside its Total. If it's all covered it should be quicker to do the example than continue this dialog. The overall figure is important and topical right now.

**Comment by ElisaKendall** [comment]
ok, but it may be awhile before I have time to do this

The approach I would take is to (1) create an individual of type UnemployedPopulation with some name that likely includes a specific date (2) add an additional type of QuantityValue, (3) add a coverage area using property hasCoverageArea and reference some jurisdiction, (4) add a reporting period using property hasReportingPeriod, (5) add a release date using hasReleaseDate or hasReleaseDateTime, and add the value (i.e. hasNumericValue).  You might need to create something that is a measurement unit or magnitude to say that the result is in millions as well, or that could be in an annotation.

**Comment by rivettp** [comment]
Reopening this since the PR did not have the approach described above i.e. an instance of UnemployedPopulation

**Reply by rivettp** [reply]
Not saying the approach is wrong - action is to prove the approach with an example.
